src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -85,6 +90,10 @@ public boolean isRunning() {
       isRunning = false;
       LOG.info("_topicManagementService not running.");
     }
+    if (!_commitAvailabilityService.isRunning()) {
```

Comment:
Why not return false when we encounter the first not-running service?

Reply:
This codestyle decision is actually a product of your suggestion back in PR number 187 (https://github.com/linkedin/kafka-monitor/pull/187)

Originally, the `isRunning` method returned false when the first service was not, in fact, running.

![image](https://user-images.githubusercontent.com/8682887/72538237-5f808580-3832-11ea-82a2-48f24471621f.png)